src/software/gui/drawing/robot.cpp

```diff
@@ -104,7 +136,7 @@ void drawRobot(QGraphicsScene* scene, const RobotStateWithId& robot, const QColo
     drawRobotAtPosition(scene, robot.robot_state.position(),
                         robot.robot_state.orientation(), color);
     drawRobotVelocity(scene, robot.robot_state.position(), robot.robot_state.velocity(),
-                      color);
+                      robot_speed_slow_color, color);
```

Comment:
Nit:

Color and colour are both used in this PR

Reply:
Yea, it's actually everywhere. Will just do a mass-renaming later at some point.